cluster_pack/spark/spark_config_builder.py

```diff
@@ -0,0 +1,39 @@
+
+import os
+
+import pyspark
+from pyspark.sql import SparkSession
+
+from cluster_pack import packaging, uploader
+
+from typing import Optional
+
+
+def add_script(ssb: SparkSession.Builder, archive: str):
+    if archive.endswith('pex'):
+        _add_or_merge(ssb, "spark.yarn.dist.files", f"{archive}")
+        ssb.config("spark.executorEnv.PEX_ROOT", "./.pex")
+        os.environ['PYSPARK_PYTHON'] = './' + archive.split('/')[-1]
+        os.environ['PYSPARK_DRIVER_PYTHON'] = archive.split('/')[-1]
+    else:
+        _add_or_merge(ssb, "spark.yarn.dist.archives", f"{archive}#condaenv")
+        os.environ['PYSPARK_PYTHON'] = f"./condaenv/bin/python"
+        os.environ['PYSPARK_DRIVER_PYTHON'] = 'python'
+
+
+def add_files(ssb: SparkSession.Builder):

```

Comment:
The name is not very intuitive, could we use something like 'add_editable_requirements' ?

Reply:
Done